src/components/project/ProjectCard.module.scss

```
@@ -1,7 +1,10 @@
 @use "../../styles/abstracts/variables" as *;
+@use "../../styles/abstracts/motion" as *;
 
 .card {
   display: flex;
+  // 썸네일 hover 확대가 둥근 모서리 밖으로 비어져 나오지 않게 잘라낸다.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)